main.c

```diff
@@ -321,7 +321,25 @@ static uint32_t patchTextureTable(Target target, uint32_t memory_offset, uint32_
     sprintf(path, "%s_%d_test.data", filename, i);
     printf("Loading '%s'\n", path);
     FILE* ft = fopen(path, "rb");
-    assert(ft != NULL);
+    
+    // Make sure the font data file actually opened and exit with an error if it didn't
+    if(ft == NULL){
+    	printf("One or more font data files are missing");
+    	
+    	#ifdef LOADER
+    	
+    	TerminateThread(target.process_information.hThread, 0);
+    	
+    	#else
+		
+		printf("\nYou should discard the exe you were attempting to patch as it is now likely broken");
+		
+		#endif
+		
+		printf("\nExiting...");
```

Comment:
This and the other instance: newline at the end, not the front.

Terminals will typically buffer the text (and not display it) until the next newline is reached.
(actually the MinGW libc never flushes properly, so text is never displayed until the buffers are full or the text is manually flushed)

Also the way it is currently done, would have the command prompt after exiting in the same line as this message.